spec/worker/gamedev_worker_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe GamedevWorker do
+  it "enqueues a Gamedev job" do
+    Sidekiq::Testing.fake!
+    expect {
+      GamedevWorker.perform_async
+    }.to change(GamedevWorker.jobs, :size).by(1)
+  end
+
+  context "with succesful response" do
+    it 'creates external posts' do
+      VCR.use_cassette("questions") do
+        Sidekiq::Testing.inline!
+          expect {
+            GamedevWorker.perform_async
+          }.to change(ExternalPost, :count).by(100)
```

Comment:
It would be helpful to have an additional test to verify that running the worker again does not create additional ExternalPosts.